server/src/main/java/org/candlepin/resource/CdnResource.java

```diff
@@ -88,36 +96,110 @@ public void delete(@PathParam("label") String label,
     @POST
     @Consumes(MediaType.APPLICATION_JSON)
     @Produces(MediaType.APPLICATION_JSON)
-    public Cdn create(
-        @ApiParam(name = "cdn", required = true) Cdn cdn,
+    public CdnDTO create(
+        @ApiParam(name = "cdn", required = true) CdnDTO cdnDTOInput,
         @Context Principal principal) {
-        Cdn existing = curator.lookupByLabel(cdn.getLabel());
+        Cdn existing = curator.lookupByLabel(cdnDTOInput.getLabel());
         if (existing != null) {
-            throw new BadRequestException(i18n.tr("A CDN with the label {0} already exists", cdn.getLabel()));
+            throw new BadRequestException(i18n.tr(
+                "A CDN with the label {0} already exists", cdnDTOInput.getLabel()));
         }
-        return cdnManager.createCdn(cdn);
+
+        Cdn cndToCreate = new Cdn();
+        this.populateEntity(cndToCreate, cdnDTOInput);
```

Comment:
We're no longer persisting the entity created here. Need to call into CdnCurator or Manager to create the instance. The latter is preferred here.

Reply:
cdnManager.createCdn() is called in the next line, inline with the translate() call.

Reply:
ack